store/sdk/src/test/java/org/apache/carbondata/sdk/file/ImageTest.java

```diff
@@ -815,4 +817,214 @@ public void binaryToCarbonWithHWD(String sourceImageFolder, String outputPath, S
     reader.close();
   }
 
+  @Test
+  public void testBinaryWithFolder() throws IOException, InvalidLoadOptionException, InterruptedException {
+    int num = 1;
+    String path = "./target/flowers";
```

Comment:
No need to write the data again and read it, it will slow down the PR builder, 
For existing testcase only, 
build 3 readers withFile, FileList, Folder. 

No need of this and below separate test case

Reply:
ok,merged